+ddos-guard.net +b2b.sibur.ru

YauhenZ opened the pull request:
ddos-guard.net,POST,ddos-guard.net,443,{rand}
ddos-guard.net,POST,185.178.208.65,443,{rand}
b2b.sibur.ru,POST,b2b.sibur.ru,443,{rand}

YauhenZ committed:
+ddos-guard.net +b2b.sibur.ru

ddos-guard.net,POST,ddos-guard.net,443,{rand}
ddos-guard.net,POST,185.178.208.65,443,{rand}
b2b.sibur.ru,POST,b2b.sibur.ru,443,{rand}

YauhenZ committed:
+nlmk.com +lukoil.com

nlmk.com,POST,nlmk.com,443,{rand}
lukoil.com,POST,lukoil.com443,{rand}

YauhenZ committed:
fix typo l97

nkahoang commented:
Hi @YauhenZ ,

Happy to add in b2b.sibur.ru in, not sure if adding DDoS-guard.net is a good idea as it's simply their public-facing site, not the actual infrastructure that supports the guarding of the other new sites.

YauhenZ commented:
> Hi @YauhenZ ,
> 
> Happy to add in b2b.sibur.ru in, not sure if adding DDoS-guard.net is a good idea as it's simply their public-facing site, not the actual infrastructure that supports the guarding of the other new sites.

agree

YauhenZ committed:
-ddos-guard.net

-ddos-guard.net

YauhenZ committed:
+vk +ok

vk.ru,POST,lukoil.ru,443,{rand}
vk.com,POST,vk.com,443,{rand}
ok.ru,POST,ok.ru,443,{rand}
